src/components/GameItem.js

```diff
@@ -5,6 +5,19 @@ import PropTypes from 'prop-types';
 
 class GameItem extends Component {
 
+  constructor(props) {
+      super(props);
+      this.state = {
+        cssType: ''
+      }
+    }
+
+  markClickedClickHandler = () => {
+    const cssType = this.props.type === "litter" ? 'spotted-litter' : 'spotted-nature';
```

Comment:
I would name this differently. We'd prefer you use a name that refers to the piece of the component being clicked.